src/mixer/mod.rs

```diff
@@ -0,0 +1,14 @@
+use std::borrow::Cow;
+
+pub mod softmixer;
+
+pub trait Mixer {
+    fn init(&mut self);
```

Comment:
Is this necessary ? Initialisation can happen when constructing the `Mixer` object